### ISSUE TITLE
Do not add head/body/html etc

### ISSUE THREAD

**Issue by octref** [Prettyhtml/prettyhtml#29]
```js
const prettyhtml = require('@starptech/prettyhtml')

const input = `
<template>
  <div id="app">
    <header class="header">
      <nav class="inner">
        <router-link to="/" exact>
          <img class="logo" src="~public/logo-48.png" alt="logo">
        </router-link>
        <router-link to="/top">Top</router-link>
        <router-link to="/new">New</router-link>
        <router-link to="/show">Show</router-link>
        <router-link to="/ask">Ask</router-link>
        <router-link to="/job">Jobs</router-link>
        <a class="github" href="https://github.com/vuejs/vue-hackernews-2.0" target="_blank" rel="noopener">
          Built with Vue.js
        </a>
      </nav>
    </header>
    <transition name="fade" mode="out-in">
      <router-view class="view"></router-view>
    </transition>
  </div>
</template>
`

console.log(prettyhtml(input))
```

I'm wondering why the result is:

```html
<html>
<head>
  <template>
    <div id="app">
      <header class="header">
        <nav class="inner">
          <router-link to="/" exact>
            <img class="logo" src="~public/logo-48.png" alt="logo">
          </router-link>
          <router-link to="/top">Top</router-link>
          <router-link to="/new">New</router-link>
          <router-link to="/show">Show</router-link>
          <router-link to="/ask">Ask</router-link>
          <router-link to="/job">Jobs</router-link>
          <a
            class="github"
            href="https://github.com/vuejs/vue-hackernews-2.0"
            target="_blank"
            rel="noopener"
          >Built with Vue.js</a>
        </nav>
      </header>
      <transition name="fade" mode="out-in">
        <router-view class="view"></router-view>
      </transition>
    </div>
  </template>
</head>
<body></body>
</html>
```

IMO the default behavior should be just formatting the input, not wrapping anything around it.

**Comment by StarpTech** [comment]
Hi @octref thanks for the report. I'm working on a much better parser which is less restrict and easier to maintain. The PR is ready https://github.com/Prettyhtml/prettyhtml/pull/30/files I will merge it in the next days and it would be great if you could test it intensively.

**Comment by StarpTech** [comment]
The new parser was released in `0.1.0`. I can't reproduce it anymore.

**Comment by octref** [comment]
Thanks for the quick fix!
